### PR TITLE
Bed cache

### PR DESCRIPTION
**Pull request by paytonagreen**
## Related to:
Resolves: #73, partially

## Description:

- Beds now have 'height', 'width', and 'unitOfMeasure' attributes
- Upon creation, beds are set by default to a size of 4ft x 4ft (added defaults to CREATE_BED_MUTATION)
- The individual Bed page now renders with data (SINGLE_BED_QUERY), grid is adjustable but not yet savable

## Tests: 
***TESTS ARE NEXT***

## Checklist:
Please put an x in each box that you have completed

- [x] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [x] I have commented my code in any hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/pages/Bed/Bed.js [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
+  if (error) return <p>{error.message}</p>
+  const { bed } = data
   return (
     <>
-      <h2>{bedName}</h2>
-      <BedBuilder />
+      <h2>{data.bedName}</h2>
+      <BedBuilder
+        height={bed.height}
+        width={bed.width}
```
Comment (capndave) [inline src/pages/Bed/Bed.js:23]: Can you please change "height" to "length"? I know that I introduced the offending term but we should prob excise it at (this) the first opportunity.

File: src/pages/Bed/BedBuilder/BedBuilder.js [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
+  MAX_HEIGHT = 20
 
-export const BedBuilder = () => {
+const BedBuilder = ({ height, width, unit }) => {
   const [measurements, setMeasurements] = useState({
-    width: DEF_WIDTH,
-    height: DEF_HEIGHT,
-    unit: "cm",
+    width,
+    height,
```
Comment (capndave) [inline src/pages/Bed/BedBuilder/BedBuilder.js:14]: see above

File: src/pages/Bed/BedBuilder/BedBuilder.js [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
+    unit,
   })
 
   const handleBedSettings = useCallback((measurement) => {
     setMeasurements(measurement)
   })
 
+  console.log(measurements.unit)
```
Comment (capndave) [inline src/pages/Bed/BedBuilder/BedBuilder.js:22]: Might wanna clean this up if you don't do so in another commit

**Review by capndave — APPROVED** [review]
This looks pretty good, check the comments. In general you can feel free to make your PRs "regular" instead of "draft", even if they're successive/have some of the same commits made to them. Generally speaking I think draft pr's are for asking questions about approaches, architectural decisions, etc before you solidify your contribution. Since you didn't put any specific questions in the comments I assume you don't have any big ones

File: src/pages/Bed/BedBuilder/BedBuilder.js (hunk 3 above)
Comment (paytonagreen) [reply src/pages/Bed/BedBuilder/BedBuilder.js:22]: Heard! I'll do it in my next commit.

File: src/pages/Bed/Bed.js (hunk 1 above)
Comment (paytonagreen) [reply src/pages/Bed/Bed.js:23]: What do you think about merging this for now, since it's functional? I can make my next task to clean this up but it will probably take some time since I set it up the same way on the backend and will need to fix that as well.

Comment (paytonagreen) [reply src/pages/Bed/Bed.js:23]: I was actually able to work this out quicker than expected!
